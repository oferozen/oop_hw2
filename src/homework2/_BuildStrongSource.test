#build a source with double edges to all other nodes

# Create a graph
CreateGraph graph1

# Add source node and four other nodes to graph1
CreateNode ns 30
CreateNode n1 1
CreateNode n2 2
CreateNode n3 3
CreateNode n4 4
AddNode graph1 ns
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Add the directed edge (ns,ns)
AddEdge graph1 ns ns

# Add the directed edge (ns,n1)
AddEdge graph1 ns n1

# Add the directed edge (ns,n2)
AddEdge graph1 ns n2

# Add the directed edge (ns,n3)
AddEdge graph1 ns n3

# Add the directed edge (ns,n4)
AddEdge graph1 ns n4

# Add the directed edge (n1,ns)
AddEdge graph1 n1 ns

# Add the directed edge (n2,ns)
AddEdge graph1 n2 ns

# Add the directed edge (n3,ns)
AddEdge graph1 n3 ns

# Add the directed edge (n4,ns)
AddEdge graph1 n4 ns

# Graph status
ListNodes graph1
ListChildren graph1 ns
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3
ListChildren graph1 n4