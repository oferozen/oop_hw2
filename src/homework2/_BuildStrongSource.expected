#build a source with double edges to all other nodes

# Create a graph
created graph graph1

# Add source node and four other nodes to graph1
created node ns with cost 30
created node n1 with cost 1
created node n2 with cost 2
created node n3 with cost 3
created node n4 with cost 4
added node ns to graph1
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Add the directed edge (ns,ns)
added edge from ns to ns in graph1

# Add the directed edge (ns,n1)
added edge from ns to n1 in graph1

# Add the directed edge (ns,n2)
added edge from ns to n2 in graph1

# Add the directed edge (ns,n3)
added edge from ns to n3 in graph1

# Add the directed edge (ns,n4)
added edge from ns to n4 in graph1

# Add the directed edge (n1,ns)
added edge from n1 to ns in graph1

# Add the directed edge (n2,ns)
added edge from n2 to ns in graph1

# Add the directed edge (n3,ns)
added edge from n3 to ns in graph1

# Add the directed edge (n4,ns)
added edge from n4 to ns in graph1

# Graph status
graph1 contains: n1 n2 n3 n4 ns
the children of ns in graph1 are: n1 n2 n3 n4 ns
the children of n1 in graph1 are: ns
the children of n2 in graph1 are: ns
the children of n3 in graph1 are: ns
the children of n4 in graph1 are: ns
